src/components/Documents/ViewDocument.tsx

```diff
@@ -15,24 +15,15 @@ interface Props {
   documentDate: string;
   documentUploader: string;
   targetUser: string;
+  resetDocumentId: ()=>null;
```

Comment:
why does this resolve as null? should it be void then?

Reply:
You're right, I didn't realize there was a difference between null, void, and undefined.